packages/inter-protocol/src/proposals/replaceElectorate.js

```
@@ -0,0 +1,229 @@
+  ).getVoterInvitations();
+  assert.equal(invitations.length, values(voterAddresses).length);
+
+  trace('Distribute invitations');
+  /** @param {[string, Promise<Invitation>][]} addrInvitations */
+  const distributeInvitations = async addrInvitations => {
+    await Promise.all(
+      addrInvitations.map(async ([addr, invitationP]) => {
+        const [voterInvitation] = await Promise.all([invitationP]);
+        trace('Sending voting invitations to', addr);
+        await reserveThenDeposit(
+          `econ committee member ${addr}`,
+          namesByAddressAdmin,
+          addr,
+          [voterInvitation],
+        );
+      }),
+    );
+  };
+
+  await distributeInvitations(zip(values(voterAddresses), invitations));
+};
+
+const startNewEconomicCommittee = async (
+  {
+    consume: { board, chainStorage, diagnostics, zoe },
+    produce: { economicCommitteeKit, economicCommitteeCreatorFacet },
+    installation: {
+      consume: { committee },
+    },
+    instance: {
+      produce: { economicCommittee },
+    },
+  },
+  { options: { committeeName, committeeSize } },
+) => {
+  const COMMITTEES_ROOT = 'committees';
+
+  trace('startNewEconomicCommittee');
+
+  trace(`committeeName ${committeeName}`);
+  trace(`committeeSize ${committeeSize}`);
+
+  const committeesNode = await makeStorageNodeChild(
+    chainStorage,
+    COMMITTEES_ROOT,
+  );
+  const storageNode = await E(committeesNode).makeChildNode(
+    sanitizePathSegment(committeeName),
+  );
+
+  const marshaller = await E(board).getPublishingMarshaller();
+
+  trace('Starting new EC Committee Instance');
+
+  const privateArgs = {
+    storageNode,
+    marshaller,
+  };
+
+  const startResult = await E(zoe).startInstance(
+    committee,
+    {},
+    { committeeName, committeeSize },
+    privateArgs,
+    'economicCommittee',
+  );
+  const { instance, creatorFacet } = startResult;
+
+  trace('Started new EC Committee Instance Successfully');
+
+  economicCommitteeKit.resolve(
```

usually needs `economicCommitteeKit.reset()` first. If it's already set and `reset()` isn't called, it fails silently. 😞